doc/source/wsrep-system-index.rst

```diff
@@ -1238,5 +1275,107 @@ is determined by bitmask:
 .. seealso:: `MySQL wsrep option: wsrep_sync_wait
              <https://galeracluster.com/library/documentation/mysql-wsrep-options.html#wsrep-sync-wait>`_
 
+.. variable:: wsrep_trx_fragment_size
+
+   :cli: ``--wsrep-trx-fragment-size``
+   :conf: Yes
+   :scope: Global, Session
+   :dyn: Yes
+   :default: 0
+
+Defines the the streaming replication fragment size. This variable is measured in the value defined by ``wsrep_trx_fragment_unit``. The minimum value is 0 and the maximum value is 2147483647.
+
+As of *Percona Server for MySQL* 8.0.26-16, you can update the variable with a `set_var hint <https://dev.mysql.com/doc/refman/8.0/en/optimizer-hints.html#optimizer-hints-set-var>`__.
+
+.. code-block:: mysql
+
+   mysql> SELECT @@@wsrep_trx_fragment_unit; SELECT @@wsrep_trx_fragment_size; 
+   +------------------------------+
+   | @@wsrep_trx_fragment_unit    |
+   +==============================+
+   | statements                   |
+   +------------------------------+
+   | @@wsrep_trx_fragment_size    |
+   +------------------------------+
+   | 3                            |
+   +------------------------------+
+   mysql> SELECT /*+ SET_VAR(wsrep_trx_fragment_size=5) */ @@wsrep_trx_fragment_size;
+   +------------------------------+
+   | @@wsrep_trx_fragment_size    |
+   +==============================+
+   | 5                            |
+   +------------------------------+
+
+You can also use set_var() in a data manipulation language (DML) statement. This ability is useful when streaming large statements within a transaction. 
+
+.. code-block:: mysql
+
+   node1> begin;
```

Comment:
Can we use uppercase for all SQL statements?
```suggestion
   node1> BEGIN;
```

Reply:
Updated.